seed_data/management/programs.json

```diff
@@ -176,7 +232,11 @@
                         "edx_course_key": "course-v1:MITx+Analog+Learning+200+Jan_2016",
                         "title": "Analog Learning 200 - January 2016",
                         "start_date": "2016-01-15T00:00:00",
-                        "enrollment_end": "2016-01-29T00:00:00"
+                        "enrollment_end": "2016-01-29T00:00:00",
+                        "course_price": {
+                            "price": 50,
```

Comment:
Sure, I can do that. But if that's the case, shouldn't `CoursePrice` have a foreign key to `Program`, instead of `CourseRun`?